content/en/guide/v10/web-components.md

```diff
@@ -72,17 +72,20 @@ function Foo() {
 
 ### Triggering custom events
```

Comment:
@developit 
I don't understand the intent of this section.
Does this section only describe custom events?

Reply:
I can see your point about generalising it to CustomEvents, although I'd keep referring to them as `built-in DOM Events` vs `standard DOM events`. To me the former sentence is more clear. The wording in this PR "normalizes standard DOM events to lowercase" is to me not as on point as the previous sentence as we strip the `on` prefix, convert the remainder to lowercase and do a `prop in dom` check to see if it is a built-in event.